snuba/consumers/consumer.py

```diff
@@ -428,7 +553,12 @@ def process_message(
 
 def process_message_multistorage(
     message: Message[MultistorageKafkaPayload],
-) -> Sequence[Tuple[StorageKey, Union[None, JSONRowInsertBatch, ReplacementBatch]]]:
+) -> Sequence[
```

Comment:
All these changes which have been made for multistorage will also be needed for the single storage